### PR TITLE
comment group & register contest

### PR DESCRIPTION
**Pull request by drowsy810301**

### CODE REVIEW

**Comment by nickelcheng** [comment]
Please comment register group button and don't comment upcoming contest register button.

**Comment by nickelcheng** [comment]
+1 besides the comment.

**Comment by drowsy810301** [comment]
fixed

File: contest/templates/contest/contestArchive.html [hunk 1]
```diff
@@ -66,7 +66,7 @@
             </button>
           </a>
           {% endif%}
-
+          <!--
```
Comment (nickelcheng) [inline contest/templates/contest/contestArchive.html]: Please don't comment here.
Comment the file ```contest/templates/contest/register.html``` line 14~38
Thanks!

**Comment by drowsy810301** [comment]
fixed

**Comment by nickelcheng** [comment]
+1, going to merge.

**Pull request merged**